multinet/multinet/db.py

```diff
@@ -103,7 +103,7 @@ def workspace_graph(workspace, name, arango=None):
 def table_fields(query, arango=None):
     """Return a list of column names for table `query.table` in workspace `query.workspace`."""
     workspace = db(query.workspace, arango=arango)
-    if workspace.has_collection(query.table):
+    if workspace.has_collection(query.table) and workspace.collection(query.table).count() > 0:
```

Comment:
I added this check here, because I was hitting cases where the `sample` variable had a value of `None`, and as such `sample.keys()` was throwing an error. I'm not sure if this introduces bugs or undesired behavior, but I haven't seen any adverse effects yet. @waxlamp PTAL.

Reply:
I was just bitten by this bug earlier today; thanks for finding and fixing it.